src/components/Profile/index.js

```diff
@@ -0,0 +1,25 @@
+import React, { useEffect } from 'react';
+import Grid from '@material-ui/core/Grid';
+import { useAuth } from '../Auth/AuthContext';
+import PersonalMenu from '../PersonalMenu';
+
+function Profile() {
+  const { authTokens } = useAuth();
+  useEffect(() => {
```

Comment:
If this is pending, let's make a separate issue to remind us to do it and remove this code block in the meantime.

Reply:
Sounds good! I created https://github.com/codebuddies/frontend/issues/100 and removed the empty `useEffect`, but we should keep the TODO comment.